romana-install/config.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Ensure stack nodes are ready for installation
+- hosts: stack_nodes
+  tasks:
+    - name: Wait for cloud-init to complete
+      wait_for: path="/var/lib/cloud/instance/boot-finished" state=present
+
+# Apply common settings to all nodes prior to installation
+- hosts: stack_nodes
+  roles:
+    - system/ubuntu
+
+# Include stack playbooks. Only the configured stack will be installed.
+- include: config.devstack.yml
```

Comment:
Ok. Only issue is that you will have an empty group definition in the include (since the group population is skipped), but the action you perform on the group does not have a 'when' clause. So, I imagine that the user will see a message about some action being performed, even though it actually isn't (due to the empty list). So during a vagrant install the user will see a message about doing some AWS action.

Might be useful then to add a 'when' also to that action in the include files.